xdsl/dialects/riscv.py

```diff
@@ -57,6 +58,14 @@
 from xdsl.utils.hints import isa
 
 
+class FastMathFlagsAttr(LLVMFastMathAttr):
```

Comment:
My understanding is that this will be disallowed in the near future, and that all attributes will be marked as final. (maybe operations too?) CC @math-fehr. I think we'll need a shared base class for fastmath attributes in some neutral location to get around this.

Reply:
I'll fix it after this is merge if you prefer, but yes the idea is to have a `FastMathAttrBase` that these operations will inherit!
Otherwise, with this current implementation, you could use the `riscv` `FastMathFlagsAttr` in LLVM and that would still work

Reply:
I can have a look after this is merged, I'll open an issue to have anyone pick it up whenever.

Reply:
#2064